resources/views/wishlist.blade.php

```diff
@@ -8,6 +8,7 @@
         <h1>This is your wishlist - hopefully you get what you wish for!</h1>
         @foreach($wishlist as $game)
         <p>{{$game['title']}}</p>
+        <x-game-info :game="$game" :shouldRemove="$shouldRemove" />
```

Comment:
@lrapp1 instead of using the game-info component here, can we make a wishlist-specific component instead that just shows the game title?

Reply:
This is done.